.github/workflows/_linux_transformers.yml

```diff
@@ -39,6 +39,8 @@ on:
         type: string
         default: 'v4.47.0'
         description: Transformers version
+  workflow_dispatch:
```

Comment:
This won't work. You will need to duplicate all inputs from workflow_call

Reply:
Never mind. Just check. I have renamed workflow_call.